wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -417,7 +417,20 @@ public final class MLSService: MLSServiceInterface {
     /// - Throws:
     ///   - MLSGroupCreationError if the group could not be created.
 
-    public func createGroup(for groupID: MLSGroupID) async throws {
+    public func createGroup(for groupID: MLSGroupID, with users: [MLSUser]) async throws {
+        guard let context else { return }
```

Comment:
Should an error be thrown here to say that the context was missing? Or passing the context would also work.

Reply:
good point. The context is passed at init, but is a weak ref. Looking at it I am not sure why it is. At first glance I don't see any retain cycle that could have happened.

I don't think we want to throw an error in that case.